.github/workflow-data/release-management/blog-post-header.tpl.md

```diff
@@ -2,7 +2,7 @@
 title: odo ${TAG_NAME} Released
 author: $GITHUB_ACTOR
 author_url: https://github.com/$GITHUB_ACTOR
-author_image_url: https://github.com/$GITHUB_ACTOR.png
+author_image_url: https://raw.githubusercontent.com/redhat-developer/odo/main/docs/website/static/img/logo.png
```

Comment:
We should probably keep `author_image_url` as it was; it is correct that it points to GitHub avatar. 

According to Docusarus [documentation](https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog) there is also an [`image`](https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog#image) field that should be used to this 

We should probably set `image` to odo logo for this template and all existing blog posts.